README.md

```diff
@@ -197,6 +204,48 @@ Delete the oldest 3 version of a package hosted in a different repo than the one
     token: ${{ secrets.GITHUB_PAT }}
 ```
 
+<br>
+
+### Delete oldest x number of versions and keeping y latest versions of a package hosted in the same repo as the workflow
+
+To delete oldest x number of versions while keeping minimum y latest versions of a package hosted in the same repo as the workflow the __package-name__, __num-old-versions-to-delete__, and __min-versions-to-keep__ inputs are required.
+
+__Example__
+
+Delete the oldest 3 version and always keep atleast 2 versions of a package hosted in the same repo as the workflow
+
+```yaml
+- uses: actions/delete-package-versions@v1
+  with:
+    package-name: 'test-package'
+    num-old-versions-to-delete: 3
+    min-versions-to-delete: 2
```

Comment:
@NamrataJha Shouldn't it be _min-versions-to-keep_ here?

Reply:
Yeah sorry my bad. Will fix the README.md